### PR TITLE
Add Flux CRD delay script.

### PR DESCRIPTION
**Pull request by dfds-pwes**
Previously a fixed timeframe delay was used but this caused failures in QA because CRDs were not fully deployed before being utilised.  This change introduces a small bash script that will cause a delay until the CRD's are in place.

### CODE REVIEW

File: test/integration/eu-west-1/k8s-qa18/services/terragrunt.hcl [hunk 1]
```diff
@@ -56,9 +56,10 @@ inputs = {
   # Platform Flux CD
   # --------------------------------------------------
 
-  // platform_fluxcd_deploy       = true
-  // platform_fluxcd_repo_name    = "platform-manifests-qa"
-  // platform_fluxcd_github_owner = "dfds"
+  platform_fluxcd_deploy       = true
+  platform_fluxcd_release_tag  = "v0.8.2"
```
Comment (abstrask) [inline test/integration/eu-west-1/k8s-qa18/services/terragrunt.hcl:60]: Don't we generally favour 'latest', i.e. not define, for QA?

File: test/integration/eu-west-1/k8s-qa19/services/terragrunt.hcl [hunk 2]
```diff
@@ -56,9 +56,10 @@ inputs = {
   # Platform Flux CD
   # --------------------------------------------------
 
-  // platform_fluxcd_deploy       = true
-  // platform_fluxcd_repo_name    = "platform-manifests-qa"
-  // platform_fluxcd_github_owner = "dfds"
+  platform_fluxcd_deploy       = true
+  platform_fluxcd_release_tag  = "v0.8.2"
```
Comment (abstrask) [inline test/integration/eu-west-1/k8s-qa19/services/terragrunt.hcl:60]: Don't we generally favour 'latest', i.e. not define, for QA?

**Review by abstrask — CHANGES_REQUESTED** [review]
LGTM, but what are your thoughts on the release tag?

File: test/integration/eu-west-1/k8s-qa19/services/terragrunt.hcl (hunk 2 above)
Comment (dfds-pwes) [reply test/integration/eu-west-1/k8s-qa19/services/terragrunt.hcl:60]: Yes.  Can I actually put latest there though or do I just omit the value entirely?

**Pull request merged**